api_docs/integrations-overview.md

```diff
@@ -67,3 +67,12 @@ us](/help/contact-support) and we'll see what we can do.
   something that isn't there check out Zulip's
   [REST endpoints](https://github.com/zulip/zulip/blob/main/zproject/urls.py)
   or [contact us](/help/contact-support) and we'll help you out.
+
+## Related articles
+
+* [Integrations overview](/help/integrations-overview)
```

Comment:
Is this linking to itself?

Reply:
oh, sorry about that! Not sure how that happened.